### DISCUSSION TITLE
Removable devices not available on Steam Deck

### DISCUSSION THREAD

**Discussion by Sarlochin** [Heroic-Games-Launcher/HeroicGamesLauncher#1143] [Q&A] [answered]
When choosing other install locations for the game, the SD card on the Steam Deck is not accessible / viewable.
On other location /run/ there is no folder /media/ where the SD card would normally be.
When you type the location manually i.e. /run/media/[card]/Heroic the path within the Heroic launcher are displayed correctly, however the actual install location remains on /home/deck...
I am not terribly accustomed to an Arch Linux, so there might be something obvious I am missing here.
Maybe someone here has an idea what I am doing wrong.

**Comment by flavioislima** [comment, marked as answer]
Flatpaks have limited access by design.
You can install flatseal to add the paths you want heroic to be able to access.
There is a good video on how to use heroic on the deck on the gaming on Linux YouTube channel and another on the Linux experiment channel.

**Reply by Sarlochin** [reply]
Thank you very much for that fast and helpful reply. Now everything works flawlessly.
